src/components/Dashboard/TalkDashboard.vue

```diff
@@ -509,4 +529,17 @@ function scroll({ direction }: { direction: 'backward' | 'forward' }) {
 	gap: 4px;
 	align-items: center;
 }
+
+@container talk-dashboard (max-width: 500px) {
+	@media (max-width: 600px) {
```

Comment:
Can we use our default breakpoint size for `@media` and something like "default breakpoint - navigation" for container?

If every adaptive place has its own breakpoint position, everything changers layout at a different moment, making more intermediate states.

Of, if there were specific reasons to use exactly 500/600, let's add a comment, so we know in which case we should change it.